wifiphisher/pywifiphisher.py

```diff
@@ -135,6 +141,21 @@ def check_args(args):
             '-' +
             W +
             '] Pre-shared key must be between 8 and 63 printable characters.')
+    if args.handshakecapture and not args.presharedkey:
+        sys.exit(
+            '[' +
+            R +
+            '-' +
+            W +
+            '] handshake-capture should be used with the presharekey option.')
```

Comment:
I am not sure I understand this branch here. Why `handshake-capture` and `presharedkey` options need to be set together?

A user will set the `handshake-capture` to verify the captured PSK from a scenario (e.g. the firmware-upgrade or the wifi-connect).